sympy/concrete/summations.py

```diff
@@ -483,7 +483,10 @@ def is_convergent(self):
             return S.false
 
         ### ------------- integral test -------------- ###
-        if is_decreasing(sequence_term, interval):
+        inequality = Derivative(sequence_term).doit() <= 0
+        solution_inequality = solveset(inequality, sym, S.Reals)
+        condition2 = solution_inequality.is_Interval and solution_inequality.right == upper_limit
+        if is_decreasing(sequence_term, interval) or condition2:
```

Comment:
OK, looking at the function again I see that the condition we have is one where there is a finite increase in the domain followed by a decrease from there to infinity. In this case it indicates convergence. I think that the use of `is_decreasing` is redundant because your test (I believe) includes that result. Rather than seeing if the function is always descreasing, you check to see that it is decreasing towards the upper limit of the function from some point; "always decreasing" is a subset of "decreasing somewhere".

Something that worries me a bit is how the domain returned doesn't match the given domain in `solveset`:

```
>>> is_decreasing(1/x,Interval(1,oo))
True
>>> (1/x).diff()<=0
-1/x**2 <= 0
>>> sol = solveset(_, x, Interval(1,oo))
>>> sol
(-oo, oo)
```

I expected sol to be `[1, oo)`.